src/Command/Redis/Container/ContainerInterface.php

```diff
@@ -18,8 +18,8 @@ interface ContainerInterface
      * Creates Redis container command with subcommand as virtual method name
      * and sends a request to the server.
      *
-     * @param $subcommandID
-     * @param $arguments
+     * @param        $subcommandID
+     * @param        $arguments
```

Comment:
Let's use the type that the php documentation suggests here.

Reply:
Done